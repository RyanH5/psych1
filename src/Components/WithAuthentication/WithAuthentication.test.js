import React from 'react';
import { shallow } from 'enzyme';
import WithAuthentication from './WithAuthentication';


describe('WithAuthentication', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<WithAuthentication />)

    expect(wrapper).toMatchSnapshot();
  });
});