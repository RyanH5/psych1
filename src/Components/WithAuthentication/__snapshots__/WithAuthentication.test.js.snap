// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithAuthentication should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <withAuthentication />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): [Function],
  Symbol(enzyme.__nodes__): Array [
    [Function],
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
